manifest.json

```diff
@@ -1506,6 +1506,16 @@
                 }
             },
             "versions": {
+                "1.0.3": {
```

Comment:
Assembly Attribute Version is still on 1.0.0, while the NeosMod version is 1.0.2